src/Shaders/ShadersInclude/reflectionFunction.fx

```diff
@@ -1,7 +1,7 @@
 ﻿vec3 computeReflectionCoords(vec4 worldPos, vec3 worldNormal)
 {
 #if defined(REFLECTIONMAP_EQUIRECTANGULAR_FIXED) || defined(REFLECTIONMAP_MIRROREDEQUIRECTANGULAR_FIXED)
-	vec3 direction = normalize(vDirectionW);
+	vec3 direction = vDirectionW;

```

Comment:
Why this change?

Reply:
It's redundant. All other places vDirectionW is set it's set with a normalized value.

Reply:
ok cool!